Do no use -isystem for Boost library
If installed, MacPorts Boost headers are found
--- client/dump/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ client/dump/CMakeLists.txt	2019-09-17 19:53:52.000000000 -0700
@@ -28,7 +28,7 @@
     -DBOOST_SYSTEM_NO_DEPRECATED)
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 SET(MYSQLPUMP_LIB_SOURCES
   abstract_chain_element.cc
--- mysys_ssl/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ mysys_ssl/CMakeLists.txt	2019-09-17 20:05:34.000000000 -0700
@@ -26,7 +26,7 @@
   SET(MYSYS_AES_IMPLEMENTATION my_aes_openssl.cc)
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 SET(MYSYS_SSL_SOURCES
     crypt_genhash_impl.cc
--- plugin/keyring_udf/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ plugin/keyring_udf/CMakeLists.txt	2019-09-17 20:06:04.000000000 -0700
@@ -20,7 +20,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 MYSQL_ADD_PLUGIN(keyring_udf
                  keyring_udf.cc
--- sql/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ sql/CMakeLists.txt	2019-09-17 20:06:58.000000000 -0700
@@ -24,7 +24,7 @@
 INCLUDE_DIRECTORIES(
   ${CMAKE_SOURCE_DIR}/libbinlogevents/include
 )
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 MY_INCLUDE_SYSTEM_DIRECTORIES(ICU)
 
--- unittest/gunit/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ unittest/gunit/CMakeLists.txt	2019-09-17 20:08:07.000000000 -0700
@@ -198,7 +198,7 @@
 )
 
 # Some tests require Boost.
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 MY_CHECK_CXX_COMPILER_FLAG("-fno-builtin-memcmp" HAVE_NO_BUILTIN_MEMCMP)
 
--- unittest/gunit/keyring/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ unittest/gunit/keyring/CMakeLists.txt	2019-09-17 20:08:25.000000000 -0700
@@ -25,7 +25,7 @@
   RETURN()
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 INCLUDE_DIRECTORIES(
   ${GMOCK_INCLUDE_DIRS}
